### DISCUSSION TITLE
How to properly quit an application.

### DISCUSSION THREAD

**Discussion by geppi** [isl-org/Open3D#5725] [Q&A]
I wrote an app based on the example vis_gui.py.
If I try to quit with gui.Application.instance.quit() the application process does not exit.
Also if the last window of an application is closed with gui.Window.close() the app should exit.
This does not work. The windows disappear but I have to press Ctrl-C in the terminal to terminate the application process.

**Comment by nickponline** [comment]
Press `q`

**Comment by geppi** [comment]
I have no idea how I can engage `gui.Application.instance.quit()` or `gui.Window.close()` to

> Press `q`

![1f609](https://user-images.githubusercontent.com/7973615/205884261-cea6f37d-1770-453b-8b9c-8435f9226167.png)
